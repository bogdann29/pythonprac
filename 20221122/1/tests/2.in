0+t+"!DZ@RZe&q/Te&`/G[meB-nc?0c=0tgjVJ%D%1CTWWah("=L[mv=2g,tjk!W:D[A45yNO3UYW62TB,Wq/@NC9jNJQa3%8