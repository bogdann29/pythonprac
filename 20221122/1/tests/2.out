0!D"=%8&`+"+t,W-n/G0c1C3U5y62=0=2@NA4C9D%D[J%JQL[NORZTBTWTeW:WaYWZ@[ma3c?e&eBg,h(jNjVk!mvq/q/tgtj